src/test_fixture/world.rs

```diff
@@ -42,20 +43,22 @@ impl Debug for TestStep {
             TestStep::Mul1(v) => write!(f, "TestStep/Mul1[{}]", v),
             TestStep::Mul2 => write!(f, "TestStep/Mul2"),
             TestStep::Reshare(v) => write!(f, "TestStep/Reshare[{}]", v),
+            TestStep::Shuffle => write!(f, "TestStep/Shuffle"),
```

Comment:
This will move to some other place based on review feedback. Just leaving it as a placeholder code.